test-it/ViaTests/CopyAlignmentError.via

```diff
@@ -0,0 +1,3586 @@
+// Autogenerated Vireo assembly file
```

Comment:
I would recommend adding a printf statement at the end of the first clump in the toplevel VI that just prints a string and check for that in the .vtr file. With the behavior differences between Debug and Release now and in the future and native vs JS (especially JS), there is a good chance that the Vireo may exit prematurely on error condition in the middle of executing/parsing a program without printing anything to console. A printf at the end is a safer bet, I think.

Reply:
Good point.  I think we'd still notice the crash, but this is good just to be safe.
Done.